students/hyeonwoo/users/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+import json
+from django.core.exceptions import ValidationError
+
+from django.shortcuts   import render
+from django.http        import JsonResponse
+from django.views       import View
+
+from .validation        import validation_email, validation_password
+from .models            import User
```

Comment:
1. 사용 안 하는 package는 제거해 주세요.
2. import 는 크게 3가지로 구분 지어서 작성합니다.
- 파이썬 내장 package
- pip 로 설치한 외부 package
- 프로젝트 내에 작성한 package

```suggestion
import json

from django.core.exceptions import ValidationError
from django.http            import JsonResponse
from django.views           import View

from .validation import validation_email, validation_password
from .models     import User
```